_posts/en/newsletters/2019-11-27-newsletter.md

```diff
@@ -0,0 +1,301 @@
+      #60][news60 16248].
+
+    - *GUI improvements:* graphical users can now create new wallets
+      for use with multiwallet mode from the GUI's *file* menu (see
+      [Newsletter #63][news63 new wallet]).  The GUI also now provides
+      users with [bech32][topic bech32] Bitcoin addresses by default,
+      but users can easily request a backwards-compatible P2SH-P2WPKH
+      address by toggling a checkbox next to the button to generate an
+      address (see [Newsletter #42][news42 core gui bech32]).
+
+    - *Optional privacy-preserving address management:* a new
+      `avoid_reuse` wallet flag, which can be toggled using a new
+      `setwalletflag` RPC, can be enabled to prevent the wallet from
+      spending bitcoins received to an address that was previously
+      used (see [Newsletter #52][news52 avoid_reuse]).  This prevents
+      certain privacy leaks based on block chain analysis such as [dust
+      flooding][].
+
+    For a full list of notable changes, links to the PRs where those
+    changes were made, and additional information useful to node
+    operators, please see the Bitcoin Core project's [release notes][notes
+    0.19.0].
+
+- **New LND mailing list and new host of existing mailing lists:** a [new
+  mailing list][lnd engineering] hosted by Google Groups was announced
+  for LND application developers, with an [initial post][osuntokun lnd
+  plans] by Olaoluwa
+  Osuntokun describing short-term goals for LND's next release.
+  Separately, the existing mailing lists for [Bitcoin-Dev][] and
+  [Lightning-Dev][] have recently had their hosting [transferred][togami
+  ml update] to Oregon State University's [Open Source Lab][osl] (OSL), a
+  well-respected organization that provides hosting to a variety of open
+  source projects.  Optech extends our thanks to everyone involved in
+  maintaining all of Bitcoin's many open communication channels, without
+  which this newsletter wouldn't exist.
+
+- **Schnorr/Taproot updates:** participants in the [taproot review
+  group][] have been continuing their review of the proposed soft fork
+  changes to Bitcoin, with many interesting questions being asked and
+  answered in the [logged][tbr log] ##taproot-bip-review IRC chatroom on
+  the Freenode network.  Additionally, some participants have been
+  writing their own implementations of parts of the BIPs, including for
+  the libbitcoin and bcoin full verification nodes.
+
+    Two informative blog posts were also posted this week related to the
+    security of multiparty schnorr signatures.  Blockstream engineer
+    Jonas Nick [describes][nick musig] the [MuSig][] multiparty
+    signature scheme designed to allow users of [bip-schnorr][] to
+    aggregate multiple public keys into a single pubkey that they can
+    later sign for using a single signature generated collaboratively
+    among them.  Nick describes the three steps of the MuSig signing
```

Comment:
I found this sentence a bit too long to parse easily ("that they can later sign for using..." was a bit tricky). Consider splitting it in two:

Blockstream engineer Jonas Nick describes the MuSig multiparty signature scheme. This scheme allows users of bip-schnorr to aggregate multiple public keys into a single pubkey and later sign for that aggregated pubkey using a single signature generated collaboratively among them.